pep-0517.txt

```diff
@@ -206,34 +207,42 @@ metadata directly.
   def build_wheel(wheel_directory, config_settings, metadata_directory=None):
       ...
 
-Must build a ``.whl`` file, and place it in the specified
-``wheel_directory``.
+Must build a .whl file, and place it in the specified ``wheel_directory``.
 
-If the build frontend has previously called ``get_wheel_metadata`` and
+If the build frontend has previously called ``prepare_wheel_metadata`` and
 depends on the wheel resulting from this call to have metadata
 matching this earlier call, then it should provide the path to the
 previous ``metadata_directory`` as an argument. If this argument is
 provided, then ``build_wheel`` MUST produce a wheel with identical
 metadata. The directory passed in by the build frontend MUST be
-identical to the directory created by ``get_wheel_metadata``,
+identical to the directory created by ``prepare_wheel_metadata``,
 including any unrecognized files it created.
 
+This must return the basename (not the full path) of the ``.whl`` file it
+creates, as a unicode string.
+
 Mandatory.
 
 ::
 
-  def export_sdist(sdist_directory, config_settings):
+  def build_sdist(sdist_directory, config_settings):
       ...
 
-Must export an unpacked source distribution into the specified
+Must build a .tar.gz source distribution and place it in the specified
 ``sdist_directory``.
 
-An unpacked source distribution (sdist) consists of a directory called
+A .tar.gz source distribution (sdist) contains a single top-level directory called
 ``{name}-{version}`` (e.g. ``foo-1.0``), containing the source files of the
 package. This directory must also contain the
 ``pyproject.toml`` from the build directory, and a PKG-INFO file containing
 metadata in the format described in
-`PEP 345 <https://www.python.org/dev/peps/pep-0345/>`_.
+`PEP 345 <https://www.python.org/dev/peps/pep-0345/>`_. Although historically
+zip files have also been used as sdists, this hook should produce a gzipped
+tarball. This is already the more common format for sdists, and having a
+consistent format makes for simpler tooling.
```

Comment:
Since it isn't otherwise specified, this should mandate the use of the UTF-8 based tar `PAX_FORMAT`: https://docs.python.org/3/library/tarfile.html#tarfile.PAX_FORMAT

The encoding should also be explicitly set to `UTF-8`.